docs/pydbus.rst

```diff
@@ -0,0 +1,91 @@
+
+* Support for type hints: Use Python type hints from :mod:`dasbus.typing` to define DBus types.
+
+* Generating XML specifications: Automatically generate XML specifications from Python classes
+  with the :func:`dbus_interface <dasbus.server.interface.dbus_interface>` decorator.
+
+* Support for DBus structures: Represent DBus structures (dictionaries of variants) by Python
+  objects. See the :class:`DBusData <dasbus.structure.DBusData>` class.
+
+* Support for groups of DBus objects: Use DBus containers from :mod:`dasbus.server.container`
+  to publish groups of Python objects.
+
+* Composition over inheritance: The library follows the principle of composition over
+  inheritance. It allows to easily change the default behaviour.
+
+* Lazy DBus connections: DBus connections are established on demand.
+
+* Lazy DBus proxies: Attributes of DBus proxies are created on demand.
+
+
+What is different
+-----------------
+
+* No context managers: There are no context managers in dasbus. Context managers and event
+  loops don't work very well together.
+
+* No auto-completion: There is no support for automatic completion of DBus names and paths.
+  We recommend to work with constants defined by classes from :mod:`dasbus.identifier`
+  instead of strings.
+
+* No unpacking of variants: The dasbus library doesn't unpack variants by default. It means
+  that values received from DBus match the types declared in the XML specification. Use the
+  :func:`get_native <dasbus.typing.get_native>` function to unpack the values.
+
+* Obtaining proxy objects: Call the :meth:`get_proxy <dasbus.connection.MessageBus.get_proxy>`
+  method to get a proxy of the specified DBus object.
+
+* No single-interface view: DBus proxies don't support single-interface views. Use the
+  :class:`InterfaceProxy <dasbus.client.proxy.InterfaceProxy>` class to access a specific
+  interface of a DBus object.
+
+* Higher priority of standard interfaces: If there is a DBus interface in the XML specification
+  that redefines a member of a standard interface, the DBus proxy will choose a member of the
+  standard interface. Use the :class:`InterfaceProxy <dasbus.client.proxy.InterfaceProxy>` class
+  to access a specific interface of a DBus object.
+
+* No support for help: Members of DBus proxies are created lazily, so the build-in ``help``
+  function doesn't return useful information about the DBus interfaces.
+
+* Watching DBus names: Use :class:`a service observer <dasbus.client.observer.DBusObserver>`
+  to watch a DBus name.
+
+* Acquiring DBus names: Call the :meth:`register_service <dasbus.connection.MessageBus.register_service>`
+  method to acquire a DBus name.
+
+* Providing XML specifications: Use the ``__dbus_xml__`` attribute to provide the XML
+  specification of a DBus object. Or you can generate it from the code using the
+  :func:`dbus_interface <dasbus.server.interface.dbus_interface>` decorator.
+
+* No support for polkit
```

Comment:
```suggestion
* No support for polkit.
```

Reply:
Also I wonder why do you have polkit here? What is the reason to have polkit support? System services?

Reply:
Pydbus has support for polkit for some reason, so it should be documented that dasbus does not.

Reply:
Btw. it is not a sentence, so I don't think that it should end with a dot.

Reply:
My inner writer wants to complain that you must have all items of a bullet list be sentences or not consistently. Thus the argument would be invalid. But you don't want to listen to that voice too much. It won't settle for anything less than _perfection_ :-/

Reply:
Ok, I have changed this item.